source/gui/widgets/skeletons/text_editor.cpp

```diff
@@ -1456,6 +1456,10 @@ namespace nana{	namespace widgets
 					select_.ignore_press = (arg_focus::reason::mouse_press == arg.focus_reason);
 				}
 			}
+
+			// Reset the selection as it is sticky in the "double click" path.
+			if (!arg.getting)
+				select(false);
```

Comment:
That's what the first commit did. However, it is then possible to select a word from several `textbox` controls on a single form. That looks a bit weird (and that not the way things work when you select text by dragging the mouse).